pkgs/tools/networking/saldl/default.nix

```diff
@@ -25,8 +24,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     pkg-config
-    wafHook
-    python3
+    waf.hook
+    waf.python3
```

Comment:
Why isn't `waf.python3` included in `waf.hook`?

Reply:
It is.

That's just because I am being overly cautious here.

Reply:
I was thinking this `python3` was here by other reasons besides `waf`.